traffic_ops/traffic_ops_golang/deliveryservice/deliveryservices.go

```diff
@@ -110,6 +110,42 @@ func (ds *TODeliveryService) IsTenantAuthorized(user *auth.CurrentUser) (bool, e
 	return isTenantAuthorized(ds.ReqInfo, &ds.DeliveryServiceV4)
 }
 
+const getTLSVersionsQuery = `
```

Comment:
well I can switch to using `ARRAY_AGG` if you think that's betetr, but the two can't be literally the same string because they have different `WHERE` clauses that can't be consolidated.

Reply:
In that case, I would split out the "base" query string for reuse, then just have the specific `WHERE` clauses added on in each place it's used.